build_tools/cmake/iree_copts.cmake

```diff
@@ -184,3 +184,15 @@ list(APPEND IREE_COMMON_INCLUDE_DIRS
   ${PROJECT_BINARY_DIR}/build_tools/third_party/tensorflow
   ${PROJECT_BINARY_DIR}/build_tools/third_party/tensorflow/tensorflow/compiler/mlir/hlo/include/
 )
+
+#-------------------------------------------------------------------------------
+# Third party: mlir-emitc
+#-------------------------------------------------------------------------------
+
+if(IREE_ENABLE_EMITC)
+  list(APPEND IREE_COMMON_INCLUDE_DIRS
+    ${PROJECT_SOURCE_DIR}/third_party/mlir-emitc/include
+    ${PROJECT_BINARY_DIR}/third_party/mlir-emitc/include
+  )
+  add_definitions(-DIREE_HAVE_EMITC_DIALECT)
```

Comment:
Should this just be named the same (`IREE_ENABLE_EMITC`)?

Reply:
@benvanik Proposed to name it `IREE_HAVE_EMITC_DIALECT` within a discussion on Discord, but I can rename if desired.

Reply:
If the discussion has already been had, then let's not have it again :grin: I was just wondering why the cmake variable and the macro aren't the same